nomad/operator_endpoint.go

```diff
@@ -316,8 +316,9 @@ func (op *Operator) SchedulerSetConfiguration(args *structs.SchedulerSetConfigRe
 	} else if respErr, ok := resp.(error); ok {
 		return respErr
 	}
-
-	// Check if the return type is a bool
+	// Unless this is a CAS request, Updated is always true at this point.
+	reply.Updated = true
+	// Check if the return type is a bool and optionally overwrite Updated
 	// Only applies to CAS requests
```

Comment:
The comments can be condensed a bit - it doesn't need to describe what the code does but the context for it:

```suggestion
	//  If CAS request, raft returns a boolean indicating if the update was applied
	// otherwise, assume success
	reply.Updated = true
```